src/ApiTrait.php

```diff
@@ -34,16 +34,16 @@
     public function __dispatch(SystemStrand $strand, $key, $value)
```

Comment:
This method is not declaring any return type, yet it is now returning something.
Not all paths specifically return anything.

Reply:
Good call, updated.